src/main/routes.ts

```diff
@@ -37,17 +37,19 @@ export class Routes {
     app.get(COOKIES_URL, errorHandler(new CookiesGetController().get));
     app.get(ACCESSIBILITY_STATEMENT_URL, errorHandler(new AccessibilityStatementGetController().get));
 
+    app.locals.steps = sequence;
```

Comment:
is there a need for this?

Reply:
This is so that we don't need to reload all of the step files for the check your answers page.